client/lang/src/en.json

```diff
@@ -8,5 +8,7 @@
     "CampaignAdmin.NO_CHANGES": "No changes",
     "CampaignAdmin.PUBLISHCAMPAIGN": "Publish campaign",
     "CampaignAdmin.REMOVED": "Removed",
-    "CampaignAdmin.REVERTCAMPAIGN": "Revert"
+    "CampaignAdmin.REVERTCAMPAIGN": "Revert",
+    "CampaignAdmin.FORBIDDEN": "403 Access Forbidden",
+    "CampaignAdmin.PAGE_NOT_FOUND": "404 Page not found"
```

Comment:
Since this is campaign specific, I would suggest having a more user friendly error message like:
- The campaign you are looking for can not be found.
- You do not have access to view this campaign.

Reply:
DONE